ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/traffic-policies.md

```diff
@@ -61,6 +63,60 @@ interface Management1
 
 # Interfaces
 
+## Ethernet Interfaces
+
+### Ethernet Interfaces Summary
+
+#### L2
+
+| Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
+| --------- | ----------- | ---- | ----- | ----------- | ----------- | ------------- |
+
+*Inherited from Port-Channel Interface
+
+#### Aegis Traffic-Policies
+
+| Interface | Input Traffic-Policy | Output Traffic-Policy |
+| --------- | -------------------- | --------------------- |
+| Ethernet1 | BLUE-C1-POLICY | BLUE-C2-POLICY |
+
+
+### Ethernet Interfaces Device Configuration
+
+```eos
+!
+interface Ethernet1
+   no switchport
+   traffic-policy input BLUE-C1-POLICY
+   traffic-policy output BLUE-C2-POLICY
+```
+
+## Port-Channel Interfaces
+
+### Port-Channel Interfaces Summary
+
+#### L2
+
+| Interface | Description | Type | Mode | VLANs | Native VLAN | Trunk Group | LACP Fallback Timeout | LACP Fallback Mode | MLAG ID | EVPN ESI |
+| --------- | ----------- | ---- | ---- | ----- | ----------- | ------------| --------------------- | ------------------ | ------- | -------- |
+
+#### Aegis Traffic-Policies
```

Comment:
This should not be called Aegis. Just Traffic-policies. I think it would be beneficial to move the traffic-policy-interface sections down under traffic policy documentation, to make it easier to get a complete overview.
You could collapse them to a single table for all interface types.

Reply:
Fixed!